src/pages/LetterStoragePage/components/StorageSendLetter.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { css } from '@emotion/react';
+import { useQueryClient, useMutation } from '@tanstack/react-query';
+import { isAxiosError } from 'axios';
+import LetterAccordion from '@/components/LetterAccordion';
+import LetterCard from '@/components/LetterCard';
+import TagList from '@/components/TagList';
+import { MoreHorizontal, Copy, TrashCan } from '@/assets/icons';
+import Dropdown from '@/components/Dropdown';
+import COLORS from '@/constants/colors';
+import PolaroidModal from '@/components/PolaroidModal';
+import { SendLetter } from '@/types/letter';
+import letterAPI from '@/api/letter/apis';
+import ERROR_RESPONSES from '@/constants/errorMessages';
+import letterOptions from '@/api/letter/queryOptions';
+import { getTagList } from '../utils/tagUtills';
+import StorageContent from './StorageContent';
+interface StorageSendLetterProps {
+  letters: SendLetter[];
+}
```

Comment:
p5;
앗!! 사소하지만.. 요 부분 한 칸 띄어 쓰는 것도 좋을 것 같아요!

```suggestion
import StorageContent from './StorageContent';

interface StorageSendLetterProps {
  letters: SendLetter[];
}
```